YouTubeSpammerPurge.py

```diff
@@ -526,13 +526,13 @@ def confirm_continue(message=""):
   # While loop until valid input
   valid = False
   while valid == False:
-    response = input("\n" + message + " (y/n): ")
-    if response == "Y" or response == "y":
+    response = input("\n" + message + " (yes continue/no stop): ")
+    if response == "Yes keep going" or response == "Sure":
       return True
```

Comment:
There is no reason to make people type out a long entry

Reply:
"Sure Yes keep going" is the first thing I type when I want to continue😆😆

Reply:
yeah exactly, we're assuming people using this program are YouTubers, and YouTubers have no time to waste, they have a busy schedule. so this just waste peoples time.

Reply:
Did you see the best part though?
The prompt is `(yes continue/no stop)` So the user has no way to know the correct reply without reading the code.
Until they try that is, then `("\nInvalid Input. Enter Yes keep going or No stop ")` comes up.

Reply:
> Did you see the best part though? The prompt is `(yes continue/no stop)` So the user has no way to know the correct reply without reading the code. Until they try that is, then `("\nInvalid Input. Enter Yes keep going or No stop ")` comes up.

lol